src/neuroglancer/ui/annotations.ts

```diff
@@ -757,7 +757,7 @@ export class AnnotationLayerView extends Tab {
       }
       annotationString += ',';
       if (annotation.description) {
-        annotationString += annotation.description;
+        annotationString += '"' + annotation.description.trim() + '"';
```

Comment:
Double quotes are only used if the string contains a comma, newline or double quote. Otherwise your double quotes will be interpreted as part of the original description.
Also, each double quote inside the description needs to be escaped by another double quote.

`This description is fine.` --> `This description is fine.`
`This description is "fine".\n` --> `"This description is ""fine"".\n"`